webapp/src/monacoFlyout.tsx

```diff
@@ -79,6 +81,15 @@ export class MonacoFlyout extends React.Component<MonacoFlyoutProps, MonacoFlyou
         return () => { this.setState({ selectedBlock: name != this.state.selectedBlock ? name : undefined }) };
     }
 
+    protected getBlockMouseOver = (name: string) => {
+        pxt.tickEvent("monaco.toolbox.itemmouseover", undefined, { interactiveConsent: true });
```

Comment:
This tick should probably be in the returned fn? right now it sends the tick for every block in the flyout each time the flyout component re-renders - so opening flyout without hovering or hovering over any single snippet would send it for each.

Also, should this one count as interactiveConsent? I suppose it probably doesn't matter at the moment as I don't think we have a monaco `flyoutOnly` mode so they'd have to have clicked on the toolbox to get here, but I wouldn't think of hover as counting as consent generally

Reply:
Yeah, don't pass interactive consent here. It needs to be a click on an interactive element in the page or a link